packages/core/test/unit/query-generator/show-tables-query.test.ts

```diff
@@ -0,0 +1,64 @@
+import { expectsql, createSequelizeInstance } from '../../support';
+
+const sequelize = createSequelizeInstance();
+const queryGenerator = sequelize.getQueryInterface().queryGenerator;
+
+const sequelizeWithSchema = createSequelizeInstance({
+  schema: 'schema_at_init',
+});
+const queryGeneratorWithSchema
+  = sequelizeWithSchema.getQueryInterface().queryGenerator;
+
+describe('QueryGenerator#showTablesQuery', () => {
+  it('should use the schema from initialization options', async () => {
+    expectsql(queryGeneratorWithSchema.showTablesQuery(), {
+      postgres: `SELECT table_name FROM information_schema.tables WHERE table_schema = 'schema_at_init' AND table_type LIKE '%TABLE' AND table_name != 'spatial_ref_sys';`,
+      mssql:
+        'SELECT TABLE_NAME, TABLE_SCHEMA FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_TYPE = \'BASE TABLE\';',
+      // TODO fix TABLE_SCHEM typo and missing quotes for CURRENT SCHEMA
```

Comment:
One quick comment on this; I believe that this is actually correct. `TABLE_SCHEM` does give results when looking through the documentation and `CURRENT SCHEMA` is also valid since it's not a specific schema name but a keyword

Reply:
Thank you 🙂 

Removed the TODO comment